3p/bodymovinanimation.js

```diff
@@ -67,14 +72,19 @@ export function bodymovinanimation(global) {
   const dataReceived = parseJson(global.name)['attributes']._context;
   const dataLoop = dataReceived['loop'];
   const animatingContainer = global.document.createElement('div');
+  setStyles(animatingContainer, {
```

Comment:
This is needed for the HTML container - it needs an already sized element.

Reply:
also works without impact on svg renderer I assume since it is not guarded by renderer type?